src/Microsoft.DotNet.HelixPoolProvider.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="$(MicrosoftAzureKeyVaultVersion)" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
+    <PackageReference Include="Microsoft.Data.OData" Version="5.8.4" />
```

Comment:
Is it worth finding where this dependency comes from and updating that package before adding the explicit reference? This is fine, but if we can just update an existing reference I'd prefer that.

Reply:
in this case it comes from Microsoft.DotNet.Helix.JobSender, I think adding the package in this case is the right thing to do, do you think that a different update can be done?

Reply:
Yeah. I think we should update the dependency in the Helix.JobSender, which comes from arcade, and unblock https://github.com/dotnet/arcade-pool-provider/pull/540 so that we get the new jobsender.

Reply:
https://github.com/dotnet/arcade/pull/7953/files already updated this in arcade for the JobSender. We should just unblock the arcade update now.

Reply:
I took a look and Microsoft.DotNet.Helix.JobSender already has the right version, so unblocking #540 should solve this, 😄 working on that